src/cpp/CMakeLists.txt

```diff
@@ -184,6 +186,9 @@ endif()
 # core library
 add_subdirectory(core)
 
+# test executable
+add_subdirectory(tests/cpp)
```

Comment:
I think we should use the RSTUDIO_UNIT_TESTS_ENABLED around this call rather than inside the tests/cpp/CMakeLists.txt